kms/api/CloudKmsTest/CloudKmsTest.cs

```diff
@@ -24,7 +29,17 @@ namespace GoogleCloudSamples
     // </summary>
     public class CommonTests
     {
-        private static readonly string s_projectId = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
+        private static readonly string s_projectId = LoadProjectId();
+
+        private static string LoadProjectId()
+        {
+            string projectId = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");

```

Comment:
Create a constant for the name of the env variable, since you are using it more than once.

Reply:
Done